internal/provider/workspace_cli_resource.go

```diff
@@ -387,13 +387,13 @@ func (r *WorkspaceCliResource) Delete(ctx context.Context, req resource.DeleteRe
 	workspaceCliRequest.Header.Add("Authorization", fmt.Sprintf("Bearer %s", r.token))
 	workspaceCliRequest.Header.Add("Content-Type", "application/vnd.api+json")
 	if err != nil {
-		resp.Diagnostics.AddError("Error creating team resource request", fmt.Sprintf("Error creating team resource request: %s", err))
+		resp.Diagnostics.AddError("Error creating vcs resource request", fmt.Sprintf("Error creating vcs resource request: %s", err))
```

Comment:
vcs? cli, I presume

Reply:
This is just a typo error in the workspace_cli_resource